stable/harbor/questions.yaml

```diff
@@ -0,0 +1,577 @@
+  label: Redis Volume Size
+  group: "Redis Settings"
+  show_if: "redis.type=internal&&persistence.enabled=true"
+- variable: persistence.persistentVolumeClaim.redis.existingClaim
+  default: ""
+  description: "If not empty, uses the specified existing PVC instead of creating new one"
+  type: pvc
+  label: Existing Persistent Volume Claim for Redis
+  group: "Redis Settings"
+  show_if: "redis.type=internal&&persistence.enabled=true"
+- variable: redis.external.addr
+  required: true
+  default: ""
+  description: "Redis address for redis: <host_redis>:<port_redis>, or for redis+sentinel: <host_sentinel1>:<port_sentinel1>,<host_sentinel2>:<port_sentinel2>,<host_sentinel3>:<port_sentinel3>"
+  type: string
+  label: External Redis Address
+  show_if: "redis.type=external"
+  group: "Redis Settings"
+- variable: redis.external.sentinelMasterSet
+  default: ""
+  description: "The name of the set of Redis instances to monitor, it must be set to support redis+sentinel"
+  type: string
+  label: External Redis Sentinel Master Set
+  show_if: "redis.type=external"
+  group: "Redis Settings"
+- variable: redis.external.password
+  default: ""
+  description: "External redis server password"
+  type: password
+  label: External Redis password
+  show_if: "redis.type=external"
+  group: "Redis Settings"
+# trivy options
+- variable: trivy.enabled
+  default: true
+  description: "Enable Trivy for vulnerability static analysis of containers images"
+  type: boolean
+  label: Enable Trivy
+  group: "Trivy Settings"
+- variable: persistence.persistentVolumeClaim.trivy.storageClass
+  default: ""
+  description: "If undefined or null, uses the default StorageClass. Default to null"
+  type: storageclass
+  label: Default StorageClass for Trivy
+  show_if: "trivy.enabled=true&&persistence.enabled=true"
+  group: "Trivy Settings"
+- variable: persistence.persistentVolumeClaim.trivy.size
+  required: true
+  default: "5Gi"
+  description: "Trivy Persistent Volume Size"
+  type: string
+  label: Trivy Volume Size
+  group: "Trivy Settings"
+  show_if: "trivy.enabled=true&&persistence.enabled=true"
+- variable: persistence.persistentVolumeClaim.trivy.existingClaim
+  default: ""
+  description: "If not empty, uses the specified existing PVC instead of creating new one"
+  type: pvc
+  label: Existing Persistent Volume Claim for Trivy
+  group: "Trivy Settings"
+  show_if: "trivy.enabled=true&&persistence.enabled=true"
+- variable: trivy.gitHubToken
+  default: ""
+  description: "A GitHub access token used to download the Trivy DB without incurring rate limiting"
+  type: string
+  label: Set a GitHub access token for Trivy
+  show_if: "trivy.enabled=true"
+  group: "Trivy Settings"
+# notary options
+- variable: notary.enabled
+  default: true
+  description: "Enable Notary for publishing and managing trusted collections of content like Docker images"
+  type: boolean
+  label: Enable Notary
+  group: "Notary Settings"
+- variable: core.replicas
```

Comment:
I'm not sure if those commented options (like `#notary options` above) have some other meaning than just comments, but here you're starting new group without a comment

Reply:
just comments, but fixed